modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/components/forms/ErrorFeedback.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayForm from '@clayui/form';
+import React from 'react';
+
+const ErrorFeedback = ({message: message}: {message: string}) => (
+	<ClayForm.FeedbackGroup>
+		<ClayForm.FeedbackItem className="font-weight-normal">
+			<ClayForm.FeedbackIndicator symbol="info-circle" />
+
+			<strong>{Liferay.Language.get('error')}: </strong>
```

Comment:
Better if this is reviewed by design, but I think that for input (or any form field) validations, we should not show the Error text, just directly the error message